validator/docs/schemas.md

```diff
@@ -0,0 +1,192 @@
+    schedule: '* * * * *' # STRING - Required
+    suspend:              # BOOLEAN
+  another-resource:
+    schedule: '* * * * *'
+```
+
+**_Note_**: You can read more about the concurrency policies in this section of the [Kubernetes documentation](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#concurrency-policy)
+
+## Kubernetes Configuration 
+
+### Scales
+When you deploy an application in a Kubernetes environment, you define how many replicas you would like to run. 
+Each replica represents a Kubernetes Pod that encapsulates your application container. 
+You can also increase, or reduce the number of replicas in the given limit.
+
+```yaml
+scales:
+  my-process:         # Process to scale
+    min-replicas: 1   # Minimal number of pods, INTEGER - Required
+    max-replicas: 10  # Minimal number of pods, INTEGER - Required
+    targetCPUUtilizationPercentage: 80 # Percentage of CPU which will trigger a horizontal scaling when it is exceeded, INTEGER, 0 <= x <= 100 - Required
+  another-process: 
+    min-replicas: 1
+    max-replicas: 5
+    targetCPUUtilizationPercentage: 95
+```
+
+### Resources
+This structure defines the resources to be allocated for your application.
+
+```yaml
+resources:
+  my-resource:  # Name of the resource to define
+    limit:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    limits:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    request:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    requests:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+  another-resource: ...
+```
+
+
+### TemplateVars
+
+Kubernetes has service settings to allow to expose an application running on a set of Pods as a network service. The idea is that Kubernetes handles the network layer to provide IP addresses to your Pods and a single DNS name for a set of Pods and can load-balance across them. You can read more in [the official Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/).
+
+In this section you can define variables used in configuration templates when building the kubernetes configuration before deployment. Here are the available variables:
+
+**_Note_**: All the variables should be prefixed by `tpl`. To avoid colliding with the default values for your Kubernetes Namespace.
+
+```yaml
+templateVars:         # Variables used in conf templates
+  tplCriticity:       # Define if the app is on the critical path, STRING - high, low, none
+  tplPublicServices:  # Define if the application web is public accessible or not, BOOLEAN
+  tplExpandedTimeout: # Define the timeout on the ingress controller, STRING
+  tplSessionAffinity: # Activate or deactivate session affinity on ingress controller, BOOLEAN
+  tplTerminationGracePeriod: # Define POD termination grace period, INTEGER
+  tplCanary:          # Define name for canary, STRING
+```
+
+### Dependencies
```

Comment:
I am not sure this part is useful and if we keep it, it should not be under Kubernetes, I think